src/components/themed/ExchangeQuoteComponent.tsx

```diff
@@ -99,8 +105,20 @@ export const ExchangeQuote = (props: Props) => {
           )}
         </View>
       )
+    } else if (quote.minReceiveAmount != null) {
+      // Show the minimum receive amount
+      return (
+        <View style={styles.bottomContainer}>
+          {renderRow(
+            <EdgeText style={styles.bottomText}>{lstrings.swap_minimum_receive_amount}</EdgeText>,
+            <EdgeText style={styles.bottomText}>{minCryptoAmountText}</EdgeText>
+          )}
+          {renderRow(<></>, <EdgeText style={styles.bottomText}>({minFiatAmountText})</EdgeText>)}
+        </View>
+      )
+    } else {
```

Comment:
If the previous condition has a `return`, you don't logically need an `else`. This would apply not only to the `if (quote.minReceiveAmount != null)` block, but also to the `else { return null }` block below.

Reply:
Yeah I know it works either way, this makes it clearer (at least to me) that these are 3 paths in the logic at this point. 

When I read an if statement in the middle of a function, I would normally have to examine the rest above to make sure there isn't some dependent stuff or early returns between the various if blocks or which if block returns.

If I see chains of else ifs or switches I easily know there's no funny business in between the blocks

Reply:
Personally I always get blind-sighted by early returns